.github/workflows/build_and_test.yml

```diff
@@ -180,6 +180,8 @@ jobs:
       run: rustup toolchain install nightly --component rustfmt --component clippy --allow-downgrade
     - name: Add no_std toolchain
       run: rustup toolchain install nightly-x86_64-unknown-linux-gnu ; rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
+    - name: Use nigntly
+      run: rustup default nightly    
```

Comment:
Let's not default to nightly, well not find a range of bugs

Reply:
ok